README.md

```diff
@@ -128,6 +108,18 @@ cd rap-package-template
 python examples/example_create_publication.py
 ```
 
+## Using GitHub codespaces
+
+If you are using GitHub Codespaces, the above installation steps will be completed automatically, so you don't need to do anything! 
+
+Click the "Code" button above, click the "Codespaces" tab, and then click the "+" button to create a new codespace. The environment may take a minute or two to build when you load it for the first time.
+
+## Running the pipeline
+
+To run the pipeline, enter the following command into the terminal. Make sure you are in the same folder as the `create-publication.py` file:
+
+`spark-submit --master local create_publication.py`
```

Comment:
Providing instructions on how to make sure you are in the same folder can really help new users get going quick!

Reply:
Added a line for new users to change the directory